src/Caliburn.Micro.Core/Screen.cs

```diff
@@ -103,20 +103,33 @@ async Task IActivate.ActivateAsync(CancellationToken cancellationToken)
 
             if (!IsInitialized)
             {
+                try
+                {
+                    Log.Info("Initializing {0}.", this);
 #pragma warning disable CS0618 // Type or member is obsolete
-                await OnInitializeAsync(cancellationToken);
+                    await OnInitializeAsync(cancellationToken);
 #pragma warning restore CS0618 // Type or member is obsolete
-                IsInitialized = initialized = true;
-                await OnInitializedAsync(cancellationToken);
+                    IsInitialized = initialized = true;
+                    await OnInitializedAsync(cancellationToken);
+                }
+                catch (Exception ex)
+                {
+                    Log.Error(ex);
```

Comment:
Missing call to `OnInitializeAsyncException`/`OnInitializedAsyncException`

Reply:
I agree, either all the methods should log the exception or there should be a OnException method that the user can hook into.

Reply:
fixed